backend/endpoints/registerMiddlewares.ts

```diff
@@ -24,4 +25,5 @@ export function registerMiddlewares(deps: Dependencies): void {
 			};
 		}
 	});
+	deps.application.use(cors());
```

Comment:
This has to be moved to the very beginning - cors is the first thing that should happen in the request stack, because all it does is verify that the domain where the request originated is valid. No point in doing anymore processing if we can do a cheap check at the start.